packages/core/src/api.ts

```diff
@@ -63,6 +64,7 @@ const getAdapterFromLoginConfig = (loginConfig: Readonly<InstanceElement>): Adap
 type VerifyCredentialsResult = {
     success: true
     accountId: AccountId
+    accountType: AccountType
```

Comment:
I think that returning
```suggestion
    success: true
    account: Account
```
with
```
type Account = {
    accountId: string,
    accountType: AccountType
}
```

Would probably be a better modeling, but it's a breaking change so I preferred adding to the existing one.

WDYT?

Reply:
how about this?
```typescript
type VerifyCredentialsResult = 
  ({ success: true } & Account)
  | {
    success: false
    error: Error
  }
```
not as clean as your suggestion, but this is non-breaking